common/src/main/java/com/tencent/rss/common/ShuffleDataResult.java

```diff
@@ -26,7 +26,11 @@
   private List<BufferSegment> bufferSegments;
 
   public ShuffleDataResult() {
```

Comment:
why we need ShuffleDataResult() and ShuffleDataResult(byte[] data) ?

Reply:
empty result

Reply:
Only two fields for now, use constructor is enough.

Reply:
Passing null value to constructor is weird. Is there a better solution?

Reply:
Use empty list

Reply:
OK